02-peer-protocol.md

```diff
@@ -207,12 +207,16 @@ definitions they reuse even feature bits, but they are not an
 arbitrary combination (they represent the persistent features which
 affect the channel operation).
 
-The currently defined types are:
+The currently defined basic types are:
   - no features (no bits set)
   - `option_static_remotekey` (bit 12)
   - `option_anchor_outputs` and `option_static_remotekey` (bits 20 and 12)
   - `option_anchors_zero_fee_htlc_tx` and `option_static_remotekey` (bits 22 and 12)
 
+Each basic type has the following variations allowed:
+  - `option_scid_alias` (bit 46)
+  - `option_zeroconf` (bit 48)
```

Comment:
Does the `option_zeroconf` bit depend on `option_scid_alias`? I feel like it should since you need an alias anyways, and with `option_scid_alias`, you MUST set an alias.

Reply:
I guess then you couldn't have a public zero-conf channel, but I still think the spec should say MUST set an alias if zero-conf

Reply:
You can totally have a public zero conf channel! Obviously it can't be announced until the channel confirms, but you can use it privately until then, and you can't have the SCID_alias feature on a public channel.